test/extensions/firstorder/batch_grad/batchgrad_settings.py

```diff
@@ -1,142 +1,10 @@
-"""Test configurations for `backpack.core.extensions.firstorder` for batch_grad
+"""Test configurations to test batch_grad
 
-
-Required entries:
-    "module_fn" (callable): Contains a model constructed from `torch.nn` layers
-    "input_fn" (callable): Used for specifying input function
-    "target_fn" (callable): Fetches the groundtruth/target classes 
-                            of regression/classification task
-    "loss_function_fn" (callable): Loss function used in the model
-
-Optional entries:
-    "device" [list(torch.device)]: List of devices to run the test on.
-    "id_prefix" (str): Prefix to be included in the test name.
-    "seed" (int): seed for the random number for torch.rand
+The tests are taken from `test.extensions.firstorder.firstorder_settings`, 
+but additional custom tests can be defined here by appending it to the list.
 """
 
-
-import torch
-from test.core.derivatives.utils import classification_targets, regression_targets
+from test.extensions.firstorder.firstorder_settings import FIRSTORDER_SETTINGS
 
 BATCHGRAD_SETTINGS = []
```

Comment:
A remark about naming: I would prefer if we had something like
```
SHARED_SETTINGS = FIRSTORDER_SETTINGS
LOCAL_SETTINGS = [...] # could be empty
BATCHGRAD_SETTINGS = SHARED_SETTINGS + LOCAL_SETTINGS
```
Can you impose a similar convention for all firstorder extensions in the test suite?